src/full-screen/components/full-screen-button.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from "react";
+
+import css from "./full-screen-button.scss";
+
+interface IProps {
+  isFullScreen: boolean;
+  handleToggleFullScreen: () => void
+}
+
+export const FullScreenButton: React.FC<IProps> = (props) => {
+  const {isFullScreen, handleToggleFullScreen} = props;
+  const [isHidden, setIsHidden] = useState(false);
```

Comment:
```suggestion
  const [isHintHidden, setIsHintHidden] = useState(false);
```
Just to make it clear that this doesn't refer to hiding the button itself.